helpdesk_sale_order/wizard/stock_picking_return.py

```diff
@@ -0,0 +1,49 @@
+from odoo import api, fields, models
+
+
+class ReturnPickingLine(models.TransientModel):
+    _inherit = "stock.return.picking.line"
+
+
+class ReturnPicking(models.TransientModel):
+    _inherit = "stock.return.picking"
+
+    picking_id = fields.Many2one(domain="[('id', 'in', linked_transfer_ids)]")
+
+    related_ticket_id = fields.Many2one(
+        comodel_name="helpdesk.ticket", string="Related ticket"
+    )
+
+    related_sale_order_id = fields.Many2one(
+        comodel_name="sale.order",
+        related="related_ticket_id.sale_order_id",
+        string="Related sale order",
+    )
+
+    linked_transfer_ids = fields.Many2many(
```

Comment:
should include the reference model's `_name`, ie: `linked_stock_picking_ids` or `linked_picking_ids`